packages/core/useContextMenu/component.ts

```diff
@@ -0,0 +1,46 @@
+import { computed, defineComponent, h, ref, toRefs } from 'vue-demi'
+import { onClickOutside } from '../onClickOutside'
+import type { MenuItem, MenuPosition } from '.'
+
+export const ContextMenu = defineComponent({
+  setup(props) {
+    const { items, position } = toRefs(props)
+
+    const style = computed(() => `position:fixed; top: ${position.value!.y}px; left:${position.value!.x}px;`)
+
+    const contextMenu = ref<HTMLElement>()
+
+    onClickOutside(contextMenu, () => {
+      contextMenu.value!.style.display = 'none'
+    })
+
+    return () => {
+      return h(
```

Comment:
Maybe `teleport` to body?

Reply:
Can we just give a wrap element? style is up to the developer

Reply:
I wanted the user to be able to create a context menu without using any HTML. So I don't want to use teleport. You are right about styling. Maybe it should be options like that:
```
export interface ContextMenuOptions{
  backgroundColor:string
  hoverColor:string
}
```

What do you think about this way?